openlibrary/templates/my_books/dropper.html

```diff
@@ -0,0 +1,35 @@
+$def with(page, edition_key=None, async_load=False)
+
+$if not edition_key:
+  $ edition = page if page.key.startswith("/books/") else None
+  $ edition_key = edition and edition.key
+
+$ work = page if page.key.startswith('/works/') else page.works[0] if page.works else None
```

Comment:
This may need to be updated to take into consideration works, editions, **and authors**